.github/kokoro/steps/hostsetup.sh

```diff
@@ -7,6 +7,9 @@ echo "========================================"
 echo "Host adding PPAs"
 echo "----------------------------------------"
 sudo apt-add-repository 'ppa:ubuntu-toolchain-r/test'
+sudo add-apt-repository 'deb https://apt.kitware.com/ubuntu/ xenial main'
```

Comment:
https://github.com/SymbiFlow/prjxray/blob/3749dc63c490e0064f243d4289e50ff72cbd83ad/.github/kokoro/steps/hostsetup.sh#L12-L18
```
echo
echo "========================================"
echo "Host adding PPAs"
echo "----------------------------------------"
wget -O - https://apt.kitware.com/keys/kitware-archive-latest.asc 2>/dev/null | sudo apt-key add -
sudo apt-add-repository 'deb https://apt.kitware.com/ubuntu/ xenial main'
echo "----------------------------------------"
```

Reply:
thanks Tim, this looks nicer indeed. Updated.